openslides_backend/action/generics.py

```diff
@@ -50,8 +51,8 @@ def create_action_prepare_dataset(self, payload: ActionPayload) -> DataSet:
             # Collect relation fields and also check structured relations and template fields.
```

Comment:
This comment is not up to date because the follwing code now handles not only relation fields but also other template fields.

Reply:
I think the "and template fields" includes the new changes, doesn't it?